plugin/hack.vim

```diff
@@ -102,12 +102,17 @@ function! <SID>HackPopulateQuickfix(hh_result)
     cgetexpr a:hh_result
   endif
 
+  let l:winnr = winnr()
   if g:hack#autoclose
     botright cwindow
   else
     botright copen
   endif
 
+  if (s:nvim && l:winnr != winnr())
+    exe l:winnr . "wincmd w"
```

Comment:
Apologies for the n00b question. What does this do? Since it's new code, I'm guessing it focuses back to the original window? Do we know why we're always focusing the quickfix window in the first place -- can we stop doing that, instead of focusing it and then directing focus back?

Reply:
It saves the the current location before opening the qf window and jumps back if it changes. As far as I know opening the window is like opening any other buffer, i.e. it would focus it automatically. I'm not sure why vim7 doesn't have this behavior but it's especially disruptive as an async job as you have a few seconds to keep typing only to have your cursor location moved.